ui/src/templates/api/index.ts

```diff
@@ -9,6 +9,7 @@ import {
   TemplateBase,
   Task,
   VariableTemplate,
+  Label,
 } from 'src/types'
 import {IDashboard, Cell} from '@influxdata/influx'
```

Comment:
Is it possible to leave templates files untouched? As in is it possible to not have type conflicts while still changing the rest of labels to use the new client? These files are all going to go away soon-ish and they are not particularly well tested and any changes make me nervous. Let's really really make sure that importing and exporting works as usual if you are going to change them.

Reply:
I tested and it is looking good ! but going forward if we can not touch them, it's probably best to not touch them. ?

Reply:
Definitely! I hear what you're saying and will make sure to steer clear of `templates` any and all times